Boss/Mod/ChannelCreator/Manager.cpp

```diff
@@ -68,7 +71,65 @@ Ev::Io<void> report_proposals( S::Bus& bus, char const* prefix
 namespace Boss { namespace Mod { namespace ChannelCreator {
 
 void Manager::start() {
-	bus.subscribe<Msg::Init
+    min_amount = default_min_amount;
+    max_amount = default_max_amount;
+    Ln::Amount custom_min_amount;
+    Ln::Amount custom_max_amount;
+
+    bus.subscribe<Msg::Manifestation
+            >([this](Msg::Manifestation const& _) {
+        return bus.raise(Msg::ManifestOption{
+            "clboss-min-channel-size",
+            Msg::OptionType_Int,
+            Json::Out::direct(min_amount.to_sat()),
+            "Minimum channel size CLBOSS will create. "
+            "Set to an integer number of satoshis. "
+            "default=500000"
+        });
+    });
+
+    bus.subscribe<Msg::Option
+            >([this, &custom_min_amount](Msg::Option const& o) {
+        if (o.name != "clboss-min-channel-size")
+            return Ev::lift();
+        custom_min_amount = Ln::Amount::sat((double) o.value);
+        if (custom_min_amount != min_amount) {
+            min_amount = custom_min_amount;
+            return Boss::log( bus, Info
+                              , "ChannelCreator: "
+                                "set min-channel-size: %s."
+                                , std::string(min_amount).c_str()
+                                );
+        }
+        return Ev::lift();
+    });
+    bus.subscribe<Msg::Manifestation
+    >([this](Msg::Manifestation const& _) {
+        return bus.raise(Msg::ManifestOption{
+                "clboss-max-channel-size",
+                Msg::OptionType_Int,
+                Json::Out::direct(max_amount.to_sat()),
+                "Maximum channel size CLBOSS will create. "
+                "Set to an integer number of satoshis. "
+                "default=16777215"
+        });
+    });
+    bus.subscribe<Msg::Option
+    >([this, &custom_max_amount](Msg::Option const& o) {
+        if (o.name != "clboss-max-channel-size")
+            return Ev::lift();
+        custom_max_amount = Ln::Amount::sat((double) o.value);
```

Comment:
Why not just:

```c++
max_amount = Ln::Amount::sat((double) o.value);
```

and remove the `custom_`\* variiables completely?

Also I prefer the casting style `double(o.value)` over `(double) o.value`.

Reply:
That makes a lot more sense. Now updated (and fixed indentation inside the lambdas to match your style, I think!)